Jasmine/Common/ViewControllers/GameElements/GameOverViewController.swift

```diff
@@ -0,0 +1,79 @@
+import UIKit
+
+class GameOverViewController: UIViewController {
+
+    // MARK: - Constants
+    private static let tileSpace: CGFloat = 8.0
```

Comment:
Hmm, I thought the point of a Constants file is so that we can change everything related to these settings in one file (Constants)?

I'm also OK with this approach, just clarifying